nibabel/cmdline/tests/test_utils.py

```diff
@@ -5,13 +5,16 @@
 Test running scripts
 """
 
-from numpy.testing import (assert_almost_equal,
-                           assert_array_equal)
-
-from nose.tools import (assert_true, assert_false, assert_raises,
-                        assert_equal, assert_not_equal)
+from nose.tools import assert_equal
 
+import nibabel as nib
+import numpy as np
 from nibabel.cmdline.utils import *
+from nibabel.cmdline.diff import get_headers_diff
+from os.path import (dirname, join as pjoin, abspath)
+
+
+DATA_PATH = abspath(pjoin(dirname(__file__), '../../tests/data'))
```

Comment:
there is already one defined, just use
```python
from nibabel.testing import data_path
```
note that in above '/' within the path might be *nix specific and might (?) not work on Windows.  That is why we use all the `path.join`